rxparse.py

```diff
@@ -260,6 +276,19 @@ def store_formulary(parsedformulary):
 
     return formulary
 
+RxRec = namedtuple('FormularyRecord', ['NAMEDOSE', 'BLACKLISTED', 'DOSE', 'COST', 'CATEGORY', 'SUBCATEGORY'])
+
+def expand_formulary(formulary):
+    '''Expand list of formulary record objects into useful name dose entries.
+    '''
+
+    formularyexpanded = []
+
+    for record in formulary:
```

Comment:
Get rid of the useless RxRec namedtuple and expand_formulary... It is just trash right now. If you want to experiment, do it on a topic branch.